features/txt-parfmt-props.feature

```diff
@@ -4,6 +4,17 @@ Feature: Get or set paragraph formatting properties
   I need a ParagraphFormat object with read/write formatting properties
 
 
+  @wip
+  Scenario Outline: Get tab stops object
+    Given a paragraph having <value> tab_stop elements
+     Then paragraph_format.tab_stops is a CT_TabStops object
+      
+    Examples:  values for paragraph_format.tab_stop elements 
+      | value |
+      | no    |
+      | some  |
+      
```

Comment:
Okay, this acceptance test is merged in feature/tabstops on the main repo. You should rebase onto that and you can proceed with the implementation of this bit. You can submit all those commits (there might only be one), I expect it will be close enough for me to merge with any tweaks.

I'll provide feedback on the rest now, and you can submit them in commits following the ParagraphFormat.tab_stops commit(s).